src/commands/Minion/bury.ts

```diff
@@ -0,0 +1,105 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+
+import { BotCommand } from '../../lib/BotCommand';
+import { stringMatches, formatDuration, rand } from '../../lib/util';
+import { Time, Activity, Tasks } from '../../lib/constants';
+import { BuryingActivityTaskOptions } from '../../lib/types/minions';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+import Prayer from '../../lib/skilling/skills/prayer';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { SkillsEnum } from '../../lib/skilling/types';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			altProtection: true,
+			oneAtTime: true,
+			cooldown: 1,
+			usage: '<quantity:int{1}|name:...string> [name:...string]',
+			usageDelim: ' '
+		});
+	}
+
+	async run(msg: KlasaMessage, [quantity, boneName = '']: [null | number | string, string]) {
+		if (!msg.author.hasMinion) {
+			throw `You dont have a minion`;
+		}
+
+		if (msg.author.minionIsBusy) {
+			return msg.send(msg.author.minionStatus);
+		}
+		// default bury speed
+		const speedmod = 1;
+
+		if (typeof quantity === 'string') {
+			boneName = quantity;
+			quantity = null;
+		}
+
+		const bone = Prayer.Bones.find(
+			bone =>
+				stringMatches(bone.name, boneName) ||
+				stringMatches(bone.name.split(' ')[0], boneName)
+		);
+
+		if (!bone) {
+			throw `That's not a valid bone to bury. Valid bones are ${Prayer.Bones.map(
+				bone => bone.name
+			).join(', ')}.`;
+		}
+
+		if (msg.author.skillLevel(SkillsEnum.Prayer) < bone.level) {
+			throw `${msg.author.minionName} needs ${bone.level} Prayer to bury ${bone.name}.`;
+		}
+
+		// Time to bury a bone
```

Comment:
i dont think we need a comment saying `timeToBuryABone` variable is the `time to bury a bone` 😄